tools/bundler/index.js

```diff
@@ -165,7 +165,13 @@ function getDefinitionName(def) {
   }
   if (def.startsWith('http://asyncapi.com/bindings')) {
     const result = bindingsRegex.exec(def);
-    if (result) return `${result[1].replace('/', '-')}-${result[2]}-${result[3]}`;
+    if (result) {
```

Comment:
Might need some comments here explaining what what is 😅

Reply:
Guess this is where the bug is for bindings?

Reply:
done